.github/workflows/activerecord_appraisals.yml

```diff
@@ -0,0 +1,37 @@
+name: ActiveRecord Appraisals
+
+on:
+  schedule: [ cron: '0 0 * * *' ]
+  pull_request:
+
+jobs:
+  appraise:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 1
+      matrix:
+        ruby:
+          - '3.1'
```

Comment:
Let's maybe drop 3.1 from here. Its support [ends in two weeks](https://endoflife.date/ruby), and [AR 8.0 tests](https://github.com/rage-rb/rage/actions/runs/13778681431/job/38590344365?pr=133) currently fail because of it.